gradle/libs.versions.toml

```diff
@@ -7,7 +7,7 @@ okio = "3.3.0"
 serialization = "1.4.1"
 kotlinx-datetime = "0.4.0"
 kotlinx-coroutines = "1.6.4"
-kotlin-wrappers = "1.0.0-pre.492"
+kotlin-wrappers = "1.0.0-pre.490"
```

Comment:
what's the reason?

Reply:
and btw where is CI execution in this PR 🌚

Reply:
> what's the reason?

The upgrade `org.jetbrains.kotlin-wrappers:kotlin-wrappers-bom | 1.0.0-pre.490 -> 1.0.0-pre.492` has significant changes in kotlin-react-router logic. Will skip for now. I didn't manage this upgrade, it requires refactor `routing` (as I get)

> and btw where is CI execution in this PR 🌚

it's failed is due to yarn.lock, it was updated by github and a new execution wasn't triggered